.github/instructions/config-files.instructions.md

```
@@ -0,0 +1,41 @@
+- Test with `cargo build` and `cargo test`
+
+**Python (ai-service):**
+- Use `pdm add <package>` in `ai-service/`
+- For dev dependencies: `pdm add -d <package>`
+- Test with `make test`
+
+**Frontend:**
+- Use `npm install <package>` in `frontend/`
+- For dev dependencies: `npm install -D <package>`
+- Test with `npm run build`
+
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)